### ISSUE TITLE
Volt not installing with php 8.1 valet default installation

### ISSUE THREAD

**Issue by thenibirahmed** [LaravelDaily/Larastarters#106]
Volt not installing with php 8.1 valet default installation

It's failing on npm install, build manifest not found

**Comment by krekas** [comment]
@thenibirahmed just installed volt on fresh laravel app with no errors. And `npm install` has nothing to do with manifest while installing. Using valet with php 8.1

**Comment by thenibirahmed** [comment]
@krekas 

check this private video 
[https://www.youtube.com/watch?v=n8EHYsX6RyQ](https://www.youtube.com/watch?v=n8EHYsX6RyQ)

**Comment by thenibirahmed** [comment]
[https://youtu.be/Pt-leWf75Ko](https://youtu.be/Pt-leWf75Ko)

**Comment by krekas** [comment]
@thenibirahmed with both videos I get `Video unavailable. This video is private`. If you want to share it make it unlisted not private.

**Comment by thenibirahmed** [comment]
oops
please check now

**Comment by krekas** [comment]
Well it seems the problem is on your end, because everything works for me. Try deleting `node_modules` and then running `npm install`.

**Comment by thenibirahmed** [comment]
I've done it and the results are in the 2nd video when I run npm run dev. Can you check please?

**Comment by krekas** [comment]
Doesn't seem like result is the same. When compiling this theme you get a bunch of sass warnings but it still works.

**Comment by thenibirahmed** [comment]
do you get the same warnings I got on this 2nd video?

**Comment by krekas** [comment]
Yes I get all those warnings
![image](https://user-images.githubusercontent.com/11015977/211217816-4c5b3350-7e2e-4dec-be41-753fc65e42c9.png)

But the theme still works
![image](https://user-images.githubusercontent.com/11015977/211217844-e67d5a89-a9b9-4685-8aec-2e31026b3ffa.png)

**Comment by thenibirahmed** [comment]
Thanks a lot,
At first here it was not working. How I fix was I removed the `node-sass` from `package.json` and then ran `npm install` after that I added it again and it ran well.

But it would be great if these errors were resolved too. 

Another thing that I can see is the font awesome is not working. Not on your side too. The icon beside users and two-level menu are broken. I had to add it with the CDN now.
